source/tones.py

```diff
@@ -87,3 +92,31 @@ def beep(
 	generateBeep(buf, hz, length, left, right)
 	player.stop()
 	player.feed(buf.raw)
+
+
+BeepSequenceElement = Union[int, Tuple[int, int]]  # Either delay_ms or (frequency_hz, duration_ms)
+BeepSequence = collections.abc.Iterable[BeepSequenceElement]
+
+
+def beepSequence(*sequence: BeepSequenceElement) -> None:
+	"""Play a simple synchronous monophonic beep sequence
+	A beep sequence is an iterable containing one of two kinds of elements.
+	An element consisting of a tuple of two items is interpreted as a frequency and duration. Note, this function plays beeps synchronously, unlike tones.beep
+	A single integer is assumed to be a delay in ms.
+	"""
+	for element in sequence:
+		if not isinstance(element, collections.abc.Sequence):
+			time.sleep(float(element) / 1000)
+		else:
+			freq, duration = element
+			time.sleep(float(duration) / 1000)
+			beep(freq, duration)
```

Comment:
Won't this order result in the first beep being delayed, and the function exiting prematurely after the last beep?
```suggestion
			time.sleep(float(duration) / 1000)
			beep(freq, duration)
```